entities/service.go

```diff
@@ -13,12 +13,18 @@ const (
 	PodIP        = "podip"
 	ClusterIP    = "clusteip"
 	NodePort     = "nodeport"
-	ExternalName = "externalname"
+	ServiceName  = "servicename"
 	LoadBalancer = "loadbalancer"
 
 	Allprotocols = "allprotocols"
 )
 
+type Service struct {
```

Comment:
nit: maybe change to `ServiceTemplate` to a bit differentiate from k8s service object?

Reply:
entities/service is analogy to entities/pod, as `Pod` is a struct, wanted to keep the consistency.